Core/Systems/SkillSystem/SkillPlayer.cs

```diff
@@ -1,7 +1,56 @@
-﻿namespace FunnyExperience.Core.Systems.SkillSystem
+﻿using FunnyExperience.Content.Items.Gear;
+using FunnyExperience.Content.Skills.Melee;
+using Microsoft.Xna.Framework.Input;
+using Terraria.GameInput;
+
+namespace FunnyExperience.Core.Systems.SkillSystem
 {
 	internal class SkillPlayer : ModPlayer
 	{
+		private static ModKeybind _skill1Keybind;
+		private static ModKeybind _skill2Keybind;
+		private static ModKeybind _skill3Keybind;
+		private static ModKeybind _skill4Keybind;
+		private static ModKeybind _skill5Keybind;
 		public Skill[] Skills = new Skill[5];
+		
+		public override void Load() {
+			if (Main.dedServ)
+				return;
+
+			_skill1Keybind = KeybindLoader.RegisterKeybind(Mod, "UseSkill1", Keys.Q);
+			_skill2Keybind = KeybindLoader.RegisterKeybind(Mod, "UseSkill2", Keys.W);
+			_skill3Keybind = KeybindLoader.RegisterKeybind(Mod, "UseSkill3", Keys.E);
+			_skill4Keybind = KeybindLoader.RegisterKeybind(Mod, "UseSkill4", Keys.R);
+			_skill5Keybind = KeybindLoader.RegisterKeybind(Mod, "UseSkill5", Keys.T);
+		}
+		
+		public override void ProcessTriggers(TriggersSet triggersSet) 
+		{
+			if (_skill1Keybind.JustPressed)
+			{
+				Skills[0] = new Berserk(1200, 1200, 1200, 1200, 5, GearType.Sword);

```

Comment:
Skills will need to be selectable in a UI in the future so this implementation of hardcoding here is problematic. We will likely want to place information about skill instances in the Skill handler ModPlayer and retrieve them from there here.

Reply:
Yeah this was just a hardcode test so I could ensure the skill was working as intended